src/CommandHandler.cpp

```diff
@@ -40,28 +39,9 @@ static void throwMissingKey(std::string const &Key,
 // In the future, want to handle many, but not right now.
 static int g_N_HANDLED = 0;
```

Comment:
let's just remove this as it's not being used anywhere outside of this class so really has no benefit of being there